src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -7,7 +7,7 @@ import { IChoiceGroupOptionProps } from 'office-ui-fabric-react/lib/components/C
 
 describe('<ChoiceGroup />', () => {
   let component: ShallowWrapper<ChoiceGroupProps>;
-  let labelWithSublabel: JSX.Element;
```

Comment:
Mostly for my learning, what caused you to replace `JSX.Element`s?

Reply:
@chrismohr 
JSX.Element is an alias to React.ReactElement.
However, React.ReactElement only describes react classes and stateless functional components. It does not include support for `Fragment` or `Portal`.
React.ReactNode extends React.ReactElement and includes support for Fragments and Portals.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L151

I think @unindented made these changes so our types are explicit and support Fragments.

Reply:
Cool.  Thanks for the explanation.